### PR TITLE
Added italian translation

### PR DESCRIPTION
**Pull request by ricsirigu**

### CODE REVIEW

**Review by denysdovhan — CHANGES_REQUESTED** [review]
First of all, thank you for contributing ❤️

Please, find someone who speaks Italian to review and check the correctness of your translation.

File: README-it-it.md [hunk 1]
```diff
@@ -0,0 +1,1788 @@
...
+```js
+null > 0; // false
+null == 0; // false
+
+null >= 0; // true
+```
+
+### 💡 Spiegazione:
+
+Per farla breve, se `null` che è più piccolo di `0` è `false`, allora `null >= 0` è `true`. Leggi la spiegazione approfondita per questo [qui](https://blog.campvanilla.com/javascript-the-curious-case-of-null-0-7b131644e274).
```
Comment (fmaule) [inline README-it-it.md:1576]: più piccolo -> minore?

File: README-it-it.md [hunk 2]
```diff
@@ -0,0 +1,1788 @@
...
+Solitamente, se l'operatore non riesce a confrontare i suoi operanti in modo opportuno, li converte in numeri e compara questi ultimo. Quindi ci si può aspettare il seguente comportamento: 
+
+```js
+// This is not what happens
+(null == 0 + null) == +0;
+0 == 0;
+true;
+```
+
+Invece, secondo una lettura attenta delle specifiche, la conversione a numero non avviente per l'operando che ha valore `null` o `undefined`. Quindi, se abbiamo `null` da un lato del simbolo uguale, l'altro lato deve essere `null` o `undefined` per fare in modo che venga restituito `true`. Dato che non è questo il caso, verrà restituito `false`.
```
Comment (fmaule) [inline README-it-it.md:1659]: avviene*

**Review by fmaule — CHANGES_REQUESTED** [review]
Italian native speaker here.

Added a few suggestions and corrections - otherwise LGTM, and great job with the translation! It's always quite hard to decide where to translate a technical term in a different language. Often it's better to leave it in english because the translated version sounds absolutely awkward.

@denysdovhan thanks for maintaining this piece of art :)

**Comment by fmaule** [comment]
Just noticed that I've added the review comments in Italian - thought it would be easier to understand. 
I can translate the suggestions as well, if needed.

File: README-it-it.md [hunk 3]
```diff
@@ -0,0 +1,1788 @@
...
+[[[[[[ undefined ]]]]]] == '' // true
+```
+
+### 💡 Spiegazione:
+
+Guarda attentamente gli esempi precedenti! Il comportamento viene spiegato nella sezione [**7.2.13** Abstract Equality Comparison](https://www.ecma-international.org/ecma-262/#sec-abstract-equality-comparison) delle specifiche.
+
+## `undefined` e `Number`
+
+Se non passiamo argomenti al costruttore di `Number`, otteniamo `0`. Il valore `undefined` viene assegnato ai parametri formale quando non viene passato alcun valore come parametro, quindi possiamo aspettarci che `Number` senza parametri prenda `undefined` come valore del suo parametro. Invece quando inseriamo `undefined`, otteniamo `NaN`.
```
[the remark this answers is not in the document]
Comment (ricsirigu) [reply README-it-it.md:518]: Cosa ne pensi di 

Il valore `undefined` viene assegnato come valore di default quando non viene passato alcun valore.

File: README-it-it.md [hunk 4]
```diff
@@ -0,0 +1,1788 @@
...
+```
+
+### 💡 Spiegazione:
+
+In base alle specifiche:
+
+1. Se non viene passato alcun parametro durante l'invocazione della funzione, sia `n` valorizzato a `+0`.
+2. Altrimenti, sia `n` un ? `ToNumber(value)`.
+3. Nel caso di `undefined`, `ToNumber(undefined)` deve restituire `NaN`.
+
```
[the remark this answers is not in the document]
Comment (ricsirigu) [reply README-it-it.md:532]: Mi piace. 

Per la riga 530 cosa ne pensi di: 

"Altrimenti, `n` sarà il risultato di `ToNumber(value)`."

**Comment by ricsirigu** [comment]
> Italian native speaker here.
> 
> Added a few suggestions and corrections - otherwise LGTM, and great job with the translation! It's always quite hard to decide where to translate a technical term in a different language. Often it's better to leave it in english because the translated version sounds absolutely awkward.
> 
> @denysdovhan thanks for maintaining this piece of art :)

Thanks for your effort @fmaule !

File: README-it-it.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (fmaule) [reply README-it-it.md:518]: Mi sembra perfetto - sorry mi era sfuggita la notifica 👍  potresti anche scrivere 
`Il valore undefined viene assegnato di default quando non viene passato alcun valore.` in quanto `il valore` è sottointeso

**Review by denysdovhan — APPROVED** [review]

**Comment by denysdovhan** [comment]
Thanks for contributing!

**Pull request merged**